### PR TITLE
Task/Serction_08_ JavaScript_Refreshment

### PR DESCRIPTION
**Pull request by gozzafadillah**
- [x] Praktikum
1. Jawaban_No_1.js
2. Jawaban_No_2.md
3. Jawaban_No_3.js
4. Jawaban_No_4.js
5. Jawaban_No_5.js
6. Jawaban_No_6.js

- [x] Screenshots
Nama file Jawaban_No_Soal.png jadi nama screenshots berururtan

- [x] Summary

### CODE REVIEW

File: 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+var a = 5;
+let b = "kampus merdeka";
+const nama = "Budi";
+let terdaftar = true;
+let lengkap_arr = [a, b, nama, terdaftar];
+
+let TakeArrayIndex2 = lengkap_arr[2];
```
Comment (syukronarie) [inline 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js]: variable js pake camel case style

File: 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+var a = 5;
+let b = "kampus merdeka";
+const nama = "Budi";
+let terdaftar = true;
+let lengkap_arr = [a, b, nama, terdaftar];
```
Comment (syukronarie) [inline 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js]: variable let pake camel case style

File: 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+var a = 5;
```
Comment (syukronarie) [inline 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js]: alasan pake var?

**Review by syukronarie — COMMENTED** [review]
Please cek komen.

File: 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js (hunk 1 above)
Comment (gozzafadillah) [reply 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js]: done sudah saya ubah mas

File: 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js (hunk 2 above)
Comment (gozzafadillah) [reply 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js]: done sudah saya ubah mas

File: 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js (hunk 3 above)
Comment (gozzafadillah) [reply 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js]: done mas di ganti, saya cuma tulis ulang code dari praktikum alterra mas.

**Review by syukronarie — APPROVED** [review]
reviewed and appproved. tks

File: 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+let a = 5;
+let b = "kampus merdeka";
+const nama = "Budi";
+let terdaftar = true;
+let lengkapArr = [a, b, nama, terdaftar];
+
+let TakeArrayIndex2 = lengkapArr[2];
```
Comment (syukronarie) [inline 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js]: let pake camel case style

Comment (gozzafadillah) [reply 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js]: done mas, sorry ga keliatan tadi kepencet resolve all

Comment (gozzafadillah) [reply 08_JavaScript_Refreshment/praktikum/Jawaban_No_1.js]: done

**Pull request merged**